src/test/java/com/wikia/webdriver/common/core/imageutilities/Shooter.java

```diff
@@ -44,8 +45,15 @@ public File capturePage(WebDriver driver) {
    * @return File path  - file's handler which was saved in given path
    */
   public File captureWebElement(WebElement element, WebDriver driver) {
-    Object[] rect = getBoundingClientRect(element, driver);
-    File image = imageEditor.cropImage((Point) rect[0], (Dimension) rect[1], capturePage(driver));
+    Point start = element.getLocation();
+    Dimension size = element.getSize();
+    if (!"FF".equals(Configuration.getBrowser())) {
```

Comment:
So... basically it worked only in FF? :smiling_imp: And probably in Google Chrome when `devicePixelRatio === 1` :smile_cat: If it works now on both browsers, let's remove Notes from method description.